CMakeLists.txt

```diff
@@ -155,6 +155,29 @@ endif()
 
 target_link_libraries(TreeSheets PRIVATE ${TREESHEETS_LIBS})
 
+### Trimsheets
+
+find_package(glfw3 3.3 REQUIRED)
+find_package(OpenGL REQUIRED)
+
+add_executable(
+    trimsheets
+    src/im/main.cpp
+    thirdparty/imgui/imgui_demo.cpp
+    thirdparty/imgui/imgui_draw.cpp
+    thirdparty/imgui/imgui_tables.cpp
+    thirdparty/imgui/imgui_widgets.cpp
+    thirdparty/imgui/backends/imgui_impl_glfw.cpp
```

Comment:
@aardappel 
What would you think which backend should fit for cross-platform use the best? Have you any hints.
I would appreciate this greatly. TIA!

Reply:
Lobster (and my game engine) uses imgui extensively, and there we use the opengl3 + sdl2 backends..